Band aid

BenGitsCode opened the pull request:
- Fixes a lot of typos, broken links in CONTRIBUTE.md
 - Also made a small change to the til-template.

BenGitsCode committed:
Modify suggested example in til-template

- This makes more sense to me.
- Ben should really build a til-cli generator to make this even easier to
contribute to. That would be super.

BenGitsCode committed:
REMOVE required PR template link

- Mostly because it links to nothing, and no such template exists 😊
- This is better for now.
- I'm more interested in the CLI tool for the template and coming up with a
integration process right now. If there's something wacky in your PR we'll just
talk about it.

BenGitsCode committed:
I found a typo!

- But now that I look at this, that whole line is useless.
- I'll be right back with another commit.

BenGitsCode committed:
Change repository to section or directory

- I'm happier with this language.
- Makes more sense for now.

BenGitsCode committed:
I found another typo.

- That's not how you spell would. It's not wood. It's not woudl.
It's would.
- How much wood would a woodchuck chuck if he could chuck woudl?

BenGitsCode committed:
Remove entire extraneous section

- Minor change to the section above, removing the typo bit and
adding sparkles. Because magic.

BenGitsCode committed:
Remove all instances of the til-package settings

- This is what happens when you rip your CONTRIBUTE.md off of atom and sloppily
hack away the stuff you don't need--you miss stuff.

- You're writing your own next time, it's better.

BenGitsCode committed:
Fix another broken link and add a new issue link

- The first one because it's broken AND wrong.
- The second because I'm nice.

laurpaik-zz commented on CONTRIBUTE.md:
```diff
@@ -38,29 +37,30 @@ Before creating enhancement suggestions, please check [this list](#before-submit
+✨ Don't be! No snippet of knowledge is too small! ✨
 
-If you want to read about using TIL or developing packages in TIL, the [TIL Flight Manual](http://flight-manual.TIL.io) is free and available online. You can find the source to the manual in [TIL/flight-manual.TIL.io](https://github.com/TIL/flight-manual.TIL.io).
+Whether its opening an issue pointing out one of my broken links or typos, or just sharing some small code snippet, trick, hack or anything else--don't overthink it! If it feels incomplete we'll ask you to expound upon it. Make it, think about it tomorrow.
```
```diff
-Whether its
+Whether it's
```

This is another sentence that I had to read twice. You have, like, nested lists in it. Stick to one thing per example if possible. Suggestion:
```diff
-Whether its opening an issue pointing out one of my broken links or typos, or just sharing some small code snippet, trick, hack or anything else--don't overthink it!
+Whether it's opening an issue pointing out one of my broken links or just sharing some small code snippet--don't overthink it!
```

BenGitsCode replied:
Yeah, I've been letting too much of that sneak past review— or just not reading them out loud. -_-

Thanks for the diff suggestions!

laurpaik-zz commented on CONTRIBUTE.md:
```diff
@@ -38,29 +37,30 @@ Before creating enhancement suggestions, please check [this list](#before-submit
 ### Pull Requests
 
-* Fill in [the required template](PULL_REQUEST_TEMPLATE.md)
+* There is currently no formal pull request flow or template, I'm more concerned with making it easy and comfortable for people to share
```
```diff
-There is currently no formal pull request flow or template, I'm more concerned with making it easy and comfortable for people to share
+There is currently no formal pull request flow or template. I'm more concerned with making it easy and comfortable for people to share
```

laurpaik-zz reviewed (commented):
Couple grammar things, mostly calm down on how long your sentences are :)

BenGitsCode committed:
Address grammar and run on sentences

- Pull-request review https://github.com/BenGitsCode/today-i-learned/pull/29

- Take Lauren's suggestion to breaking up sentence.
- Fix 'it's its' typo.

BenGitsCode committed:
Remove extraneous sentence.

- This line doesn't provide any real value, especially with the previous bit
now cleaned up.

BenGitsCode committed:
Correct grammar in pull requests section

- It's okay to have multiple sentences Ben. 😬

laurpaik-zz reviewed (approved):
👍 great changes!
